### PR TITLE
docs: show how to install older versions on OS X

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

**Review by aiuto — COMMENTED** [review]
Does this make sense?
I'm not sure what previous versions are available via brew.

Do  you know who maintains the homebrew integration? I have no idea myself.

**Comment by aiuto** [comment]
Can someone who knows anything about homebrew integration look at this?

File: site/docs/install-os-x.md [hunk 1]
```diff
@@ -141,3 +141,11 @@ following command:
 ```shell
 $ brew upgrade bazel
 ```
+
+To install older bazel versions
+
+```
+brew tap bazelbuild/tap
```
Comment (davidxia) [inline site/docs/install-os-x.md:148]: > Do you know who maintains the homebrew integration? I have no idea myself.

This command clones the https://github.com/bazelbuild/homebrew-tap repo as documented [here](https://docs.brew.sh/Taps#the-brew-tap-command). This repo is in the same org as this one so I assume it's owned by the official bazel team?

File: site/docs/install-os-x.md [hunk 2]
```diff
@@ -141,3 +141,11 @@ following command:
...
+brew extract bazel bazelbuild/tap --version X.Y.Z
```
Comment (davidxia) [inline site/docs/install-os-x.md:149]: > I'm not sure what previous versions are available via brew.

[docs say](https://docs.brew.sh/Manpage#extract---version---force-formula-tap)

> Look through repository history to find the most recent version of formula and create a copy in tap

I was able to extract 4.0.0 and 0.29.0. I think what versions are available depend on the history.

```
brew extract bazel bazelbuild/tap --version 0.29.0
==> Searching repository history
==> Writing formula for bazel from revision 16d0135 to:
/opt/homebrew/Library/Taps/bazelbuild/homebrew-tap/Formula/bazel@0.29.0.rb
```

**Review by davidxia — COMMENTED** [review]
@aiuto thanks for reviewing. Answered your questions. Lmk if you have more!

**Comment by aiuto** [comment]
I have a different approach.  Put the documentation about older versions on https://github.com/bazelbuild/homebrew-tap instead of here.
We should keep this overview installation page as simple as possible and not encourage people to install older versions. 
We can eventually put a link to their example here.

**Comment by davidxia** [comment]
> I have a different approach. Put the documentation about older versions on https://github.com/bazelbuild/homebrew-tap instead of here. We should keep this overview installation page as simple as possible and not encourage people to install older versions. We can eventually put a link to their example here.

That works. [Like this](https://github.com/bazelbuild/homebrew-tap/pull/110)?

**Comment by davidxia** [comment]
closed by https://github.com/bazelbuild/homebrew-tap/pull/110

**Pull request closed without merging**